app/controllers/v0/sessions_controller.rb

```diff
@@ -64,6 +64,14 @@ def parse_date(date_string)
       nil
     end
 
+    def parse_gender(gender)
+      if gender.nil?
```

Comment:
I know we removed the rubocop for this but it could be a guard clause
```ruby
def parse_gender(gender)
  return nil unless gender
  gender[0].upcase
end
```